tests/data/cpu_template_helper/fingerprint_AMD_MILAN_4.14host.json

```diff
@@ -1,10 +1,11 @@
 {
   "firecracker_version": "1.5.0-dev",
-  "kernel_version": "4.14.318-241.531.amzn2.x86_64",
+  "kernel_version": "4.14.322-244.539.amzn2.x86_64",
   "microcode_version": "0xa0011a9",
   "bios_version": "1.0",
-  "bios_revision": "5.22",
+  "bios_revision": "0.57",
   "guest_cpu_config": {
+    "kvm_capabilities": [],
```

Comment:
could you please also update the commit to mention why the kvm_capabilities is added to the file?

Reply:
added in the commit message. It was added in PR #3967, but we haven't regathered fingerprint since then.